From 4e7ab499c557bf603f6b9b868dfbc504660eb374 Mon Sep 17 00:00:00 2001
From: Christian Stewart <christian@paral.in>
Date: Fri, 11 Mar 2022 04:40:57 -0800
Subject: [PATCH] Revert "Add new compiler hardening flags."

This reverts commit 39f2111b1d5f00206446257377dcce58cc72369f.

This commit breaks the build on some non-x86 platforms
---
 configure.ac | 2 --
 1 file changed, 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 17fb1e60..f133563b 100644
--- a/configure.ac
+++ b/configure.ac
@@ -192,8 +192,6 @@ if test "$GCC" = "yes" || test "$GCC" = "egcs"; then
 	# actually links. The test program compiled/linked includes a number
 	# of integer operations that should exercise this.
 	OSSH_CHECK_CFLAG_LINK([-ftrapv])
-	OSSH_CHECK_CFLAG_COMPILE([-fzero-call-used-regs=all])
-	OSSH_CHECK_CFLAG_COMPILE([-ftrivial-auto-var-init=zero])
     fi
 	AC_MSG_CHECKING([gcc version])
 	GCC_VER=`$CC -v 2>&1 | $AWK '/gcc version /{print $3}'`
-- 
2.35.1

